src/us_ofac.rs

```diff
@@ -0,0 +1,104 @@
+use crate::{ConfigError, Location};
+use std::collections::hash_map::Entry::{Occupied, Vacant};
+use std::collections::{HashMap, HashSet};
+
+fn get_non_passing_entries() -> HashMap<String, Option<HashSet<String>>> {
+  HashMap::from([
+    (
+      String::from("UA"),
+      Some(HashSet::from([String::from("Crimea")])),
+    ),
+    (String::from("IR"), None),
+    (String::from("SY"), None),
+    (String::from("CU"), None),
+    (String::from("KP"), None),
+  ])
+}
+
+/// Validates country code
+pub fn validate_country_code(location: &Location) -> Result<(), ConfigError> {
+  let err = Err(ConfigError::InvalidCountry);
+  match get_non_passing_entries().entry(location.clone().country_code) {
+    Vacant(_) => Ok(()),
+    Occupied(entry) => match entry.get() {
+      Some(region) => match region.get(location.region.as_str()) {
+        Some(_) => err,
+        None => Ok(()),
+      },
+      None => err,
+    },
+  }
```

Comment:
`get()` seems more readable than the Entry API here

```suggestion
  match get_non_passing_entries().get(&location.country_code) {
    Some(regions) => match regions.get(&location.region) {
        Some(_) => err,
        None => Ok(()),
    },
    None => err,
  }
```